perun-rpc/src/main/java/cz/metacentrum/perun/rpc/Api.java

```diff
@@ -698,6 +702,17 @@ private void serve(HttpServletRequest req, HttpServletResponse resp, boolean isG
 				result = caller.getVOOTManager().process(caller.getSession(), method, des.readAll());
 				if (perunRequest != null) perunRequest.setResult(result);
 				ser.write(result);
+			} else if (SCIMMANAGER.equals(manager)) {
+				// Process SCIM protocol
+				result = caller.getSCIMManager().process(caller.getSession(), method, des.readAll());
+				if (perunRequest != null) perunRequest.setResult(result);
+				if (!(result instanceof Response)) throw new InternalErrorException("SCIM manager returned unexpected result: " + result);
+				resp.setStatus(((Response) result).getStatus());
+				String response = (String) ((Response) result).getEntity();
+				PrintWriter printWriter = new PrintWriter(resp.getOutputStream());
+				printWriter.println(response);
+				printWriter.flush();
+				printWriter.close();
```

Comment:
This is not safe way to close the PrintWriter. There should be used try-with-resources method or close() should be called in finally block.